packages/react-native-renderer/src/findNodeHandle.js

```diff
@@ -15,25 +15,14 @@ import getComponentName from 'shared/getComponentName';
 import invariant from 'fbjs/lib/invariant';
 import warning from 'fbjs/lib/warning';
 
-// TODO: Share this module between Fabric and React Native renderers
-// so that both can be used in the same tree.
-
 let findHostInstance = function(fiber: Fiber): any {
   return null;
 };
 
-let findHostInstanceFabric = function(fiber: Fiber): any {
-  return null;
-};
-
 export function injectFindHostInstance(impl: (fiber: Fiber) => any) {
   findHostInstance = impl;
```

Comment:
Do these not overwrite each other if called twice?

Reply:
They're in separate bundles but I'm going to do a follow up that doesn't use injection and uses instantiation instead.